content/articles/stream-filter-in-java-using-lambda-expressions/index.md

```diff
@@ -40,10 +44,12 @@ Employee mike = new Employee("Mike J.", 113);
 List<Employee> employee = Arrays.asList(dennis, lanet, george,mike); 
 
 ```
-**Explanation**
-As demonstrated above, this function enables us to filter items of the stream that meet our provided criteria. The filter() function is used.
+#### Explanation
+
+As demonstrated above, the filter() function enables us to filter items of the stream that meet our provided criteria.

```

Comment:
The code example above doesn't demonstrate what you are talking about here. This doesn't make sense to me.